api/src/main/java/org/openmrs/module/appointments/dao/AppointmentServiceAttributeDao.java

```diff
@@ -0,0 +1,13 @@
+package org.openmrs.module.appointments.dao;
+
+import org.openmrs.module.appointments.model.AppointmentServiceAttribute;
+
+import java.util.List;
+
+public interface AppointmentServiceAttributeDao {
```

Comment:
Do we need this DAO? why not fold into AppointmentServiceDao? 
All saves, gets happen in context of an appointment service.